### PR TITLE
Button to share product is added

### PR DESCRIPTION
**Pull request by yegorsch**
It redirects to the localized version of the openfoodfacts website (I checked on English and Russian). The sharing item is just a link but I can change it to whatever text needed. 
img_ce5b0ae84e24-1
img_42310dc3514e-1

### CODE REVIEW

**Comment by teolemon** [comment]
@yegorsch can you fix the tiny merge conflict ?

**Comment by yegorsch** [comment]
Done

File: Sources/Helpers/SharingManager.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//  OpenFoodFacts
+//
+//  Created by Егор on 2/23/18.
+//  Copyright © 2018 Andrés Pizá Bückmann. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SharingManager {
```
Comment (roger-tan) [inline Sources/Helpers/SharingManager.swift]: Will be great to use the keyword final if this class should not be inherit in future.

File: Tests/ViewControllers/Products/Detail/Summary/SummaryHeaderCellControllerTests.swift [hunk 2]
```diff
@@ -79,4 +80,14 @@ class SummaryHeaderCellControllerTests: XCTestCase {
 
         expect(self.viewController.presentedViewController is ImageViewer).to(beTrue())
     }
+
+    // MARK: - didTapEditButton
+    func testDidTapEditButton() {
+        viewController.product = Product()
+        viewController.product.barcode = "123456789"
+
+        viewController.didTapEditButton(UIButton())
```
Comment (roger-tan) [inline Tests/ViewControllers/Products/Detail/Summary/SummaryHeaderCellControllerTests.swift]: There is probably a better way to unit test against this.

You should technically send an event from editButton and not call the method. If I have in mind, it should be something like

`editButton.sendActions(for: .touchUpInside)`

**Comment by roger-tan** [comment]
@yegorsch If you have few minutes. Could you resolve the conflicts, I will merge your pr after the conflicts.

**Comment by yegorsch** [comment]
@roger-tan Should be okay now.

**Pull request merged**